SSOT/UpdateCommonJobTitle2021.sql

```diff
@@ -0,0 +1,25 @@
+--Load file contents into a temp table
+Declare @JSON varchar(max)
+SELECT @JSON=BulkColumn
+FROM OPENROWSET (BULK 'C:\JB\workbc-cst\SSOT\ssot_nocs.json', SINGLE_CLOB) import

```

Comment:
In CTT, this path was `C:\src_ctt\SSOT\`. Can we bring it closer to that one? e.g. `C:\src_cst\SSOT\`.